met-web/src/components/survey/report/index.tsx

```diff
@@ -1,13 +1,8 @@
 import React from 'react';
 import SettingsForm from './SettingsForm';
-import { ReportSettingsContextProvider } from './ReportSettingsContext';
 
 const ReportSettings = () => {
-    return (
-        <ReportSettingsContextProvider>
-            <SettingsForm />
-        </ReportSettingsContextProvider>
-    );
+    return <SettingsForm />;
```

Comment:
I guess no context component is needed any longer? Great job simplifying the code.